### ISSUE TITLE
Derealization

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9622]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Derealization is an alteration in the perception of the external world, causing those with the condition to perceive it as unreal, distant, distorted or falsified. Other symptoms include feeling as if one's environment is lacking in spontaneity, emotional coloring, and depth.
(American Psychiatric Association (2004) Diagnostic and Statistical Manual of Mental Disorders DSM-IV-TR (Text Revision). American Psychiatric Association. [ISBN 0-89042-024-6])

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Kleine-Levin syndrome ORPHA 33543

PMID: 16230322 "Common symptoms were hypersomnia (100%), cognitive changes (96%, including a specific feeling of
**derealization**), eating disturbances (80%), hypersexuality (43%), compulsions (29%), and depressed mood (48%)."

PMID: 18438947 "The disease course was longer in men, in patients with hypersexuality, and when onset was after
age 20. During episodes, all patients had hypersomnia, cognitive impairment, and **derealization**; 66% had megaphagia; 53%
reported hypersexuality (principally men); and 53% reported a depressed mood (predominantly women)."

PMID: 28434769 "There was a strong correlation between the degree of hypoperfusion of the right and left parieto-temporal junctions and the intensity of the feeling of **derealization** (dreamlike, unreal self-perception) during the hypersomnolent period"

PMID: 30030664 "During sleep episodes, the patients are also troubled with one or more cognitive (such as language or memory impairment), psychiatric (such as **derealization**, apathy, depression), or behavioural dysfunctions (such as hyperphagia, hypersexuality, irritability, aggression)

PMID: 36590182 "Episodic hypersomnia is the major feature of KLS. Cognitive impairment, **derealization**, and commonly present during episodes"

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by bcoleman96** [comment]
I think this term is good, however, I think we need to create a parent term for it and other terms that deal with false perceptions of reality. "Abnormal perception of reality" was suggested as such.

**Comment by pnrobinson** [comment]
@carolina-f added as HP:5200218
